executors/readfile/README.md

```diff
@@ -27,6 +27,17 @@ testcases:
     assertions:
     - result.err ShouldBeEmpty
 ```
+```yaml
+name: TestSuite Read Yaml File
+testcases:
+  - name: Read File
+    steps:
+      - type: readfile
+        path: yourfile.yml
+        filetype: yaml
```

Comment:
filetype: no need to add a new attribute filetype. If yml or yaml, we should extract the values, as it's already done with json format.